sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/statsEstimation/AggregateEstimationSuite.scala

```diff
@@ -45,6 +45,34 @@ class AggregateEstimationSuite extends StatsEstimationTestBase with PlanTest {
   private val nameToColInfo: Map[String, (Attribute, ColumnStat)] =
     columnInfo.map(kv => kv._1.name -> kv)
 
+  test("aggregate with alias") {
```

Comment:
super nit: `SPARK-26894: propagate child stats for aliases in Aggregate`

Reply:
Done